pinot-plugins/pinot-file-system/pinot-adls/src/main/java/org/apache/pinot/plugin/filesystem/AzurePinotFS.java

```diff
@@ -107,8 +107,12 @@ public boolean doMove(URI srcUri, URI dstUri)
   }
 
   @Override
-  public boolean copy(URI srcUri, URI dstUri)
+  public boolean copyDir(URI srcUri, URI dstUri)
       throws IOException {
+    if (isDirectory(srcUri)) {
```

Comment:
Is this true? If so, add a TODO to support recursive copy?

Reply:
based on azure data lake client. and how we use it (`getReadStream`) it will throw ADLException if the path is a folder.